src/manifolds/EssentialManifold.jl

```diff
@@ -40,20 +40,14 @@ E = (R'_1)^T [T'_2 - T'_1]_{×} R'_2,
 
 where the poses of two cameras ``(R_i', T_i'), i=1,2``, are contained in the space of
 rigid body transformations $SE(3)$ and the operator $[⋅]_{×}\colon ℝ^3 \to \operatorname{SkewSym}(3)$
-denotes the matrix representation of the cross product operator. For more details see [^TronDaniilidis2017].
+denotes the matrix representation of the cross product operator. For more details see TronDaniilidis2017(@cite).
```

Comment:
```suggestion
denotes the matrix representation of the cross product operator. For more details see [TronDaniilidis:2017](@cite).
```
This mistake is present in many of other places for this citation.

Reply:
I chill carefully check this then. It is probably also wrong in the bibtex then because otherwise DocumenterCitation gets angry ;)